components/ui/WeekdaySelect.tsx

```diff
@@ -11,10 +16,10 @@ export const WeekdaySelect = (props) => {
     props.onSelect(activeDays.map((v, idx) => (v ? idx : -1)).filter((v) => v !== -1));
   }, [activeDays]);
 
-  const toggleDay = (e, idx: number) => {
+  const toggleDay = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>, idx: number) => {
```

Comment:
If this is written inline you don't have to type this

Reply:
Easier would be to do this 

```suggestion
  const toggleDay = (idx: number) => {
```

and then do the `e.preventDefault()` inline

Reply:
... or to refactor the whole thing to react-hook-form, but I understand you don't wanna do that